lib/elixir/lib/code.ex

```diff
@@ -1638,6 +1639,10 @@ defmodule Code do
     * `:ignore_module_conflict` - when `true`, does not warn when a module has
       already been defined. Defaults to `false`.
 
+    * `:infer_signatures` (since v1.18.0) - when `false`, it disables inference
+      module-local signatures used when type checking remote calls to the compiled
```

Comment:
```suggestion
    * `:infer_signatures` (since v1.18.0) - when `false`, it disables module-local
      signature inference used when type checking remote calls to the compiled
```